README.md

```diff
@@ -498,6 +498,32 @@ There are two events that we were not able to decode, for varying reasons:
 
 _Note however, that we don't have a clean way to trap the data and log it. See [Logging unmatched events](#logging-unmatched-events) for an example of how one might log such unmatched events_
 
+### Versioned events
+The below example demonstrates the addition of a `CartId` property in a newer version of `CreateCart`. It's worth noting that
+deserializing `CartV1.CreateCart` into `CartV2.CreateCart` requires `CartId` to be an optional propery or the property will
+deserialize into `null` which is an invalid state for the `CartV2.CreateCart` record in F#.
+
+```
+module CartV1 =
+    type CreateCart = { Name: string }
```

Comment:
Note default mode does not do camel case conversion so this would not yield camelCases json (would love it that convention was doc'd too!)
```suggestion
    type CreateCart = { name: string }
```